tests/features/asset_distribution/add_asset_distribution.feature

```diff
@@ -5,22 +5,19 @@ Feature: Add asset distribution through the UI
   I need to be able to add "Asset distribution" RDF entities through the UI.
 
   Scenario: "Add distribution" button should only be shown to moderators.
-    Given the following collection:
-      | title | Asset Distribution Test |
-      | logo  | logo.png                |
-    And the following solution:
+    Given the following solution:
       | title       | Asset random name                |
       | description | Some reusable random description |
-      | collection  | Asset Distribution Test          |
+    And the following collection:
+      | title      | Asset Distribution Test |
+      | logo       | logo.png                |
+      | affiliates | Asset random name       |
```

Comment:
It is an asset. The solutions are dcat:asset. But since we use the term solution, I changed it in the test.!